### ISSUE TITLE
Use of reddit API within rate limits

### ISSUE THREAD

**Issue by timwie** [timwie/spoilerwatch#29]
Since there is a limit of 600 requests per 10 minutes to the reddit API, users should be able to login with their account. If the rate limit is reached in the user-less context, they should be forced to login.

**Comment by timwie** [comment]
Watching the rate limit is a matter of monitoring and therefore depends on #15. Logging in is implemented.

**Comment by timwie** [comment]
Monitoring is not part of the launch milestone anymore, so this can be implemented in a more straightforward way and later adjusted.
